### PR TITLE
Report parsing

### PR DESCRIPTION
**Pull request by agramfort**
improves parse of folder for report

### CODE REVIEW

File: mne/report.py [hunk 1]
```diff
@@ -1864,7 +1869,8 @@ def _render_evoked(self, evoked_fname, baseline, image_format):
             has_types = []
             if len(pick_types(ev.info, meg=False, eeg=True)) > 0:
                 has_types.append('eeg')
-            if len(pick_types(ev.info, meg='grad', eeg=False)) > 0:
+            if len(pick_types(ev.info, meg='grad', eeg=False,
+                              ref_meg=False)) > 0:
```
Comment (agramfort) [inline mne/report.py:1873]: this is a bug fix for CTF data.

File: mne/report.py [hunk 2]
```diff
@@ -181,38 +181,47 @@ def _get_fname(fname):
     return fname
 
 
+def _endswith(fname, exts):
```
Comment (hoechenberger) [inline mne/report.py:184]: Good solution! 👍

**Comment by hoechenberger** [comment]
I have rebased onto `master` to resolve merge conflicts.

**Comment by hoechenberger** [comment]
@agramfort Azure still thinks we're being abusive. It's clearly trying to gaslight us! 💩

**Comment by hoechenberger** [comment]
@agramfort build artifact: https://21069-1301584-gh.circle-artifacts.com/0/dev/auto_tutorials/misc/plot_report.html#sphx-glr-auto-tutorials-misc-plot-report-py

**Comment by agramfort** [comment]
good to go from my end.

@larsoner do you want to have a look?

**Comment by hoechenberger** [comment]
So now… do we have to wait until Azure is working again, or can this be merged..?

**Comment by agramfort** [comment]
if travis is happy I would proceed ....

>

**Comment by hoechenberger** [comment]
Thanks @agramfort!

**Pull request merged**